dsp/window/window_test.go

```diff
@@ -113,31 +122,53 @@ var windowTests = []struct {
 	},
 }
 
-var gausWindowTests = []struct {
+var monoParamWindowTests = []struct {
 	name  string
-	sigma float64
+	param float64
+	win   parametricWindow
```

Comment:
`windowType`, please. We prefer tests to be more explicit.

Reply:
Are you objecting to `winType` instead of `windowType` ?  I find that needlessly pedantic since we are in the context of `window_test.`

I would prefer to not expend any further effort on the tests as they exist at this moment.  The relevant infrastructure will be remade for the change to the complex interface anyway.

Reply:
OK. Yes, that what was I was asking for. If you are going to go through this an make additional changes, you can do it then. Otherwise I'll fix it.

FTR though, calling this pedantry is unnecessary; "pedantry" and "care" are two words for the same thing through the lens of different interests. My interest is the long term success of the project and ease of maintenance since that is where I spend much of my time. Conventionally in tests we are more explicit with naming, which is why my initial comment suggested the use of the longer label.

Reply:
I am going to rewrite and simplify the entire window test suite when I change the complex interface.

Reply:
No worries.